### DISCUSSION TITLE
Reusing Prisma connection objects

### DISCUSSION THREAD

**Discussion by abathellier** [hayes/pothos#660] [Q&A] [answered]
With the Relay plugin, you can reuse a connection object: https://pothos-graphql.dev/docs/plugins/relay#reusing-connection-objects.

How to do this with the **Prisma plugin**? We have many connection types of the same Prisma type...

**Comment by hayes** [comment]
working on this in #671

**Comment by hayes** [comment, marked as answer]
Added support for sharing connection objects between Prisma connections: https://pothos-graphql.dev/docs/plugins/prisma#sharing-connections-objects
